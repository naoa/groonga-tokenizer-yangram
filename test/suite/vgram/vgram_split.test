register tokenizers/yangram

table_create vgram_words TABLE_HASH_KEY ShortText
load --table vgram_words
[
{"_key": "今日"},
{"_key": "雨だ"}
]
tokenize TokenYaVgramSplitSymbolAlpha "今日はaは雨だなb" NormalizerAuto --mode ADD
tokenize TokenYaVgramSplitSymbolAlpha "今日はaは雨だなb" NormalizerAuto --mode GET
