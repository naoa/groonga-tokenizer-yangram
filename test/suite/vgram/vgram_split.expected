register tokenizers/yangram
[[0,0.0,0.0],true]
table_create #vgram_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table #vgram_words
[
{"_key": "今日"},
{"_key": "雨だ"}
]
[[0,0.0,0.0],2]
tokenize TokenYaVgramSplitSymbolAlpha "今日はaは雨だなb" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "今日は",
      "position": 1
    },
    {
      "value": "日は",
      "position": 2
    },
    {
      "value": "はa",
      "position": 3
    },
    {
      "value": "aは",
      "position": 4
    },
    {
      "value": "は雨",
      "position": 5
    },
    {
      "value": "雨だ",
      "position": 6
    },
    {
      "value": "雨だな",
      "position": 7
    },
    {
      "value": "だな",
      "position": 8
    },
    {
      "value": "なb",
      "position": 9
    },
    {
      "value": "b",
      "position": 10
    }
  ]
]
tokenize TokenYaVgramSplitSymbolAlpha "今日はaは雨だなb" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日は",
      "position": 1
    },
    {
      "value": "aは",
      "position": 4
    },
    {
      "value": "雨だな",
      "position": 7
    },
    {
      "value": "なb",
      "position": 9
    }
  ]
]
